packages/generators/src/handlers/react.ts

```diff
@@ -25,7 +25,7 @@ export async function questions(
   });
 
   // Add react dependencies
-  self.dependencies.push("react", "react-dom");
+  self.dependencies.push("react@18.1.0", "react-dom@18.1.0");
```

Comment:
I don't think we should pin specific version numbers

Reply:
Yep, just `react@18`

Reply:
Done 👍🏻